jupyter_server_ydoc/handlers.py

```diff
@@ -329,8 +346,10 @@ async def maybe_save_document(self):
             model["format"] = file_format
             model["content"] = self.room.document.source
             self.log.debug("Saving Y document to disk: %s", file_path)
-            model = await ensure_async(self.contents_manager.save(model, file_path))
-            self.last_modified = model["last_modified"]
+            async with self.lock:
+                model = await ensure_async(self.contents_manager.save(model, file_path))
+                self.last_modified = model["last_modified"]
+            self.log.debug(f"{self.last_modified=}")
```

Comment:
The `=` syntax was introduced in Python 3.8, so you must not use it.
https://docs.python.org/3/reference/lexical_analysis.html#f-strings

```suggestion
            self.log.debug(f"last_modified={self.last_modified}")
```

Reply:
Left-over, I forced-pushed before your comment :+1: